src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.jsx

```diff
@@ -267,6 +267,7 @@ describe('VAOS <CanceledAppointmentsList>', () => {
     };
 
     mockAppointmentInfo({ va: [appointment], isHomepageRefresh: true });
+    mockFacilitiesFetch('vha_442', []);
```

Comment:
I left these out of the test that cares if the facility info isn't there, but otherwise we can mock this and save some lines in the output.